server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -627,41 +630,46 @@ public void testSelectFromTableColumns() {
-        assertEquals(false, response.rows()[7][21]); // is_nullable
-        assertEquals("stuff['level1']", response.rows()[8][11]);
-        assertEquals(false, response.rows()[8][21]); // is_nullable
-        assertEquals("stuff['level1']['level2']", response.rows()[9][11]);
-        assertEquals(false, response.rows()[9][21]); // is_nullable
-        assertEquals("stuff['level1']['level2_nullable']", response.rows()[10][11]);
-        assertEquals(true, response.rows()[10][21]); // is_nullable
+        assertEquals("age", response.rows()[0][12]); // column_name
+        assertEquals("integer", response.rows()[0][13]); // data_type
+        assertEquals(null, response.rows()[0][14]); // datetime_precision
+        assertEquals("NEVER", response.rows()[0][21]); // is_generated
+        assertEquals(false, response.rows()[0][22]); // is_nullable
+        assertEquals(32, response.rows()[0][23]); // numeric_precision
+        assertEquals(2, response.rows()[0][24]); // numeric_precision_radix
+
+        assertEquals(3, response.rows()[0][26]); // ordinal_position
+        assertEquals(defaultSchema, response.rows()[0][27]); // table_catalog
+        assertEquals("test", response.rows()[0][28]); // table_name
+        assertEquals(defaultSchema, response.rows()[0][29]); // table_schema
+
+        assertEquals("col1", response.rows()[2][12]);
+        assertEquals(false, response.rows()[2][22]);
+
+        assertEquals("col2", response.rows()[3][12]);
+        assertEquals(true, response.rows()[3][22]);
+
+        assertEquals("b", response.rows()[1][12]);
+        assertEquals(8, response.rows()[1][23]);
+        assertEquals("s", response.rows()[6][12]);
+        assertEquals(16, response.rows()[6][23]);
+        assertEquals("d", response.rows()[4][12]);
+        assertEquals(53, response.rows()[4][23]);
+        assertEquals("f", response.rows()[5][12]);
+        assertEquals(24, response.rows()[5][23]);
+        assertEquals("stuff", response.rows()[7][12]);
+        assertEquals(false, response.rows()[7][22]); // is_nullable
+        assertEquals("stuff['level1']", response.rows()[8][12]);
+        assertEquals(false, response.rows()[8][22]); // is_nullable
+        assertEquals("stuff['level1']['level2']", response.rows()[9][12]);
+        assertEquals(false, response.rows()[9][22]); // is_nullable
+        assertEquals("stuff['level1']['level2_nullable']", response.rows()[10][12]);
+        assertEquals(true, response.rows()[10][22]); // is_nullable
+
+        assertEquals(Map.of("name","stuff","path", List.of()) , response.rows()[7][11]); // column_details
+        assertEquals(Map.of("name","stuff","path", List.of("level1")) , response.rows()[8][11]); // column_details
+        assertEquals(Map.of("name","stuff","path", List.of("level1","level2")) , response.rows()[9][11]); // column_details
+        assertEquals(Map.of("name","stuff","path", List.of("level1","level2_nullable")) , response.rows()[10][11]); // column_details
```

Comment:
maybe we can switch this to use `printedTable(response.rows())` instead to improve readability? Not really related but as we already changing it... ;)